### PR TITLE
static: make examples go gettable with vanity domain

### PR DESCRIPTION
**Pull request by kortschak**
The second commit in this PR is a strawman, but I think the idea is generally sound (the "website" path element is possibly not the best, "doc", "example",...?). I did not make the same changes, for the stat article, but this can be done assuming that what I've done is actually correct and is a wanted thing.

Also not entirely sure about the licensing; the articles are CC-BY-SA, but putting that on the stand alone code would complicate the lives of people who choose to copy the code (for whatever reason), so putting the Gonum license on the example module means that the code is essentially dual licensed, CC-BY-SA if copied from the article, and Gonum BSD if copied from the module.

Please take a look.

### CODE REVIEW

File: layouts/partials/custom_head.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <meta name="go-import" content="gonum.org/v1/netlib git https://github.com/gonum/netlib">
   <meta name="go-import" content="gonum.org/v1/plot git https://github.com/gonum/plot">
   <meta name="go-import" content="gonum.org/v1/tools git https://github.com/gonum/tools">
+  <meta name="go-import" content="gonum.org/website git https://github.com/gonum/website/static/code">
```
Comment (sbinet) [inline layouts/partials/custom_head.html]: does that actually work?
I was under the impression it was expecting a repository root as a third argument.

https://golang.org/cmd/go/#hdr-Remote_import_paths

> The meta tag has the form:
> 
> ```
> <meta name="go-import" content="import-prefix vcs repo-root">
> ```

Comment (kortschak) [reply layouts/partials/custom_head.html]: Thanks, I was worried about that. I'll drop the second commit.

Comment (kortschak) [reply layouts/partials/custom_head.html]: It's a pity, it would be nice to have a clean import path for these things.

Comment (sbinet) [reply layouts/partials/custom_head.html]: yep. if we were to put some Go code in front (ie: a `net/http` router) we could control this (and fix the lingering issue of godoc redirection for _e.g.:_ `https://gonum.org/v1/gonum/mat#Matrix` (compared to _e.g._ `https://go-hep.org/x/hep/groot/rtree#Tree`) :))

**Review by sbinet — APPROVED** [review]

**Pull request merged**